locale/en/blog/vulnerability/february-2021-security-releases.md

```diff
@@ -1,36 +1,45 @@
 ---
-date: 2021-02-18T16:00:00.000Z
+date: 2021-02-23T16:00:00.000Z
```

Comment:
I am not sure how our publishing works and whether this would delay the post being live or not. Perhaps we should update to be 13:30?

Reply:
Sounds good, I'll update this.